Ticketing/Ticketing.DAL/Repositories/CartRepository.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.EntityFrameworkCore.ChangeTracking;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+using Ticketing.DAL.Contracts;
+using Ticketing.DAL.Domain;
+
+namespace Ticketing.DAL.Repositories
+{
+    public class CartRepository : IRepository<Cart>
```

Comment:
do we really need non generic implementations? Please substitute with one generic implementation instead

Reply:
done